src/redux/reducers/standardReportTables.js

```diff
@@ -2,31 +2,53 @@ import { actionTypes } from '../actions/standardReportTables'
 
 export const defaultState = {
     loading: false,
+    error: '',
     collection: [],
+    noMatches: false,
+    searchTerm: '',
 }
 
 export const standardReportTables = (
     state = defaultState,
     { type, payload } = {}
 ) => {
     switch (type) {
+        case actionTypes.STANDARD_REPORT_TABLES_SET_SEARCH_TERM:
```

Comment:
This is more a remark on the existing code style, but referring to the action types in this manner misses on one of the intended benefits of sharing action types as constants: being notified when you're using an action type that doesn't exist. You'll commonly see action types being imported and used directly (not as a collection), because that way you'll get an error if you use a type that doesn't exist.

https://stackoverflow.com/a/37054518

Reply:
But yeah, not really part of the current PR I assume, and probably already on the radar.

Reply:
👍